.rspec

```diff
@@ -1,4 +1,3 @@
 --color
```

Comment:
from PR description it looks like you want to ignore this whole file -- 100% cool me! So let's do it? And move the `random` config to `spec_helper` ?